app/javascript/components/developer/MainPage.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState} from "react";
+import { Link } from "react-router-dom";
+import { BsFacebook } from "react-icons/bs";
+import { Splide, SplideSlide,SplideTrack } from "@splidejs/react-splide";
+import "@splidejs/splide/dist/css/themes/splide-default.min.css";
+import { BsTwitter } from "react-icons/bs";
+import { BsInstagram } from "react-icons/bs";
+
+import { useSelector, useDispatch } from "react-redux";
+import { fetchDevelopers } from "../redux/developer";
+function MainPage() {
```

Comment:
- [ ] Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latests standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const MainPage = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can make a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.